aspnet/mvc/controllers/dependency-injection.rst

```diff
@@ -65,7 +65,7 @@ Once the service has been configured, running the application and navigating to
 
 .. image:: dependency-injection/_static/server-greeting.png
 
-.. tip:: See :doc:`/testing/unit-testing` to learn how `explicitly requesting dependencies <http://deviq.com/explicit-dependencies-principle>`_ in controllers makes code easier to test.
+.. tip:: See `Unit Testing <https://docs.microsoft.com/en-us/dotnet/articles/core/testing/unit-testing-with-dotnet-test>`_ to learn how to explicitly request dependencies <http://deviq.com/explicit-dependencies-principle>`_ in controllers makes code easier to test.
```

Comment:
This unit testing link should probably link to https://docs.asp.net/en/latest/mvc/controllers/testing.html instead.